spec/book_spec.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'spec_helper'
+
+describe Book do
+  before :each do
+    @book = Book.new 'Title', 'Author'
+  end
+
+  it 'It should return book title' do
+    expect(@book.title).to eq 'Title'
+  end
+  it 'It should return book author' do
+    expect(@book.author).to eq 'Author'
+  end
```

Comment:
- [ ] Same goes for this. I would recommend adding test for all the public method you have for the respective class. 

Hint: 
- Add test for `add_rental` method.